### ISSUE TITLE
Enable Guided Tour

### ISSUE THREAD

**Issue by r-brown** [paypal/gimel#187]
Hello,
Would you consider to enable [guided tour](https://github.com/Labs64/GuideChimp) to improve user experience?

**Comment by theromit** [comment]
Yes, it completely makes sense. We have been discussing some introductory videos and quick overviews so it is easy for those who are new to Gimel to be able to understand the power of the API. We will definitely check out GuideChimp. Thanks for the pointer. 

FYI: @anishanainani @Dee-Pac
